src/test/java/org/opensearch/security/privileges/SecurityIndexAccessEvaluatorTest.java

```diff
@@ -54,17 +54,36 @@ public class SecurityIndexAccessEvaluatorTest {
     private PrivilegesEvaluatorResponse presponse;
     @Mock
     private Logger log;
-
     private SecurityIndexAccessEvaluator evaluator;
-
     private static final String UNPROTECTED_ACTION = "indices:data/read";
     private static final String PROTECTED_ACTION = "indices:data/write";
+    @Mock
+    ConfigModelV7 configModelV7;
+    @Mock
+    ConfigModelV7.SecurityRoles securityRoles;// = configModelV7.getSecurityRoles();
+
+    public void setupEvaluatorWithSystemIndicesControl() {
```

Comment:
Nit: setupEvaluatorWithSystemIndicesControl is nearly identical setupEvaluatorWithoutSystemIndicesControl, take a parameter?